gTea/app/src/main/java/com/nechaieva/gtea/OrderProcessor.java

```diff
@@ -0,0 +1,64 @@
+package com.nechaieva.gtea;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class OrderProcessor {
+
+    List<String> menu;
+    final int SENSITIVITY = 3;
```

Comment:
Consider adding a one line comment per attribute, e.g., what is this sensitivity doing? is it the logging sensitivity?